theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/FocusRangeZone/FocusRangeZone.tsx

```diff
@@ -59,39 +60,20 @@ const FocusRangeZone: React.FC<{
     usePanelDragZoneGestureHandlers(layoutP, panelStuffRef),
   )
 
-  const [onMouseEnter, onMouseLeave] = useMemo(() => {
-    let unlock: VoidFn | undefined
-    return [
-      function onMouseEnter(event: React.MouseEvent) {
-        if (event.shiftKey === false) {
-          if (unlock) {
-            const u = unlock
-            unlock = undefined
-            u()
-          }
-          unlock = panelStuffRef.current.addBoundsHighlightLock()
-        }
-      },
-      function onMouseLeave(event: React.MouseEvent) {
-        if (event.shiftKey === false) {
-          if (unlock) {
-            const u = unlock
-            unlock = undefined
-            u()
-          }
-        }
-      },
-    ]
-  }, [])
-
   const isShiftDown = useKeyDown('Shift')
+  const isPointerHovering = useHover(containerNode)
+
+  useEffect(() => {
```

Comment:
If the user intends to make a new focus range, we should not highlight the panel's bounds as that makes it seem they're about to perform an action on the panel rather than the focus range.

Before:
![before](https://user-images.githubusercontent.com/593118/166224602-fd602c39-99b5-44e9-8949-d3565964ccce.gif)

After:
![after](https://user-images.githubusercontent.com/593118/166224611-8c115c57-a47e-4b9f-b96e-30b0b933bfb1.gif)